test/common/db_sandbox.js

```diff
@@ -15,10 +15,14 @@
 'use strict';
 
 const child_process = require('child_process');
+const Promise = require('bluebird');
 const rewire = require('rewire');
+const pgpLib = require('pg-promise');
 
 const database = rewire('../../db');
 
+const inTest = process.env.NODE_ENV === 'test';
```

Comment:
Why we need it? We can explicitly set `noLocking` to `true`.

Reply:
This file will eventually be removed after this milestone. So leaving its refactoring for now.